pkg/commands/config_verify.go

```diff
@@ -47,7 +47,7 @@ func (c *configCommand) executeVerify(cmd *cobra.Command, _ []string) error {
 
 		printValidationDetail(cmd, &detail)
 
-		return fmt.Errorf("the configuration contains invalid elements")
```

Comment:
You know we limited the linters inside the configuration but you still try adding more linters.

https://github.com/golangci/golangci-lint/blob/44c070af0829b1f87ecf754e2cccc4e74a22b27f/.golangci.yml#L1-L12

So what I have to do:
- accept to enable the linter, and you will continue trying to add linter despite the existing rule.
- refuse to enable the linter, and you will think that no linter can be added and that I'm a bad person.

If to keep the stability of golangci-lint tests, I have to be seen as a bad person, it's not a problem because I know it's not true, but I think this is not the right approach.

So, maybe the rule should be clarified, or we have to find another way.